.github/workflows/contributors-record.yml

```diff
@@ -0,0 +1,42 @@
+# Give your test a name!
+name: ContributorsRecord
+
+# runs daily
+on:
+  schedule:
+    - cron:  '* */23 * * *'
```

Comment:
I might be super wrong, but I think this set up will trigger the action every minute past 23 hours. I would use instead the following: `'0 23 * * *'` to trigger the action only once at 23 pm. I like using [this page](https://crontab.guru/) to check how I write my cron jobs, in case anyone reading find that useful.

Reply:
Ah, yes. Thank you for catching that. I need them to run once per day (even less would be ok too - like once in 3 days). That cronguru link is pretty great. Thank you for sharing - so with the current change '30 09 * * *' I can expect this to run at 9:30 everyday I hope.